test-common/src/main/java/com/synopsys/integration/alert/test/common/TestProperties.java

```diff
@@ -18,7 +20,11 @@
     private String propertiesLocation;
 
     public TestProperties() {
-        propertiesLocation = TestResourceUtils.DEFAULT_PROPERTIES_FILE_LOCATION;
+        try {
+            propertiesLocation = new File(TestResourceUtils.BASE_TEST_RESOURCE_DIR.getCanonicalFile(), TestResourceUtils.DEFAULT_PROPERTIES_FILE_LOCATION).getCanonicalPath();
```

Comment:
Looks like we do use it again here, I would move this to TestResourceUtils if possible